src/components/ViewScan.vue

```diff
@@ -387,6 +391,9 @@ export default {
 
 			this.$api.expose(this.resetScan);
 			this.$api.expose(this.viewer, 'viewer');
+
+			// TODO: How to reject?
```

Comment:
create an issue?

Reply:
Nah, that’s probably never gonna happen. I removed the comment 😇️